client/src/pages/EditProfile.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Popover, WhiteSpace } from "antd-mobile";
+import { MenuOutlined, EditOutlined } from "@ant-design/icons";
```

Comment:
I think we should only use material UI icons? might need to check with designer to be sure but when I asked that's what they said.
I just would like to limit the number of dependencies so try to stick to one icon library if possible.

Reply:
Yes I talked to Yu and managed to use the exact same icons now.

Reply:
There's a way to grab icons on Figma so that we can be sure we're using the correct icons as well being less dependent on external libraries - I added instructions in the README. @CeesWang let me know if you have any questions with that!